src/providers/consumerVirtualTextDocumentProvider.ts

```diff
@@ -109,8 +109,10 @@ export class ConsumerVirtualTextDocumentProvider implements vscode.TextDocumentC
         if (!this.isActive(uri)) {
             return;
         }
-        let line = `Key: ${message.key}\nPartition: ${message.partition}\nOffset: ${message.offset}\n`;
-        line = line + `Value:\n${message.value}\n\n`;
+        const key = message.deserializedKey !== null ?  message.deserializedKey : message.key;
+        const value = message.deserializedValue !== null ?  message.deserializedValue : message.value;
```

Comment:
if deserializer === none, the content should be ignored

Reply:
This logic is done in the registry.ts in deserialize function.

Reply:
yes but if deserializer is "none", it'll return null, so this piece of code will fallback to returning message.key/value

Reply:
if format == 'none' is the same thing when key-format or value-format is not declared, so it display the received value.

Reply:
Ok now I return an empty string when format=='none'